drivers/adc/adc_stm32.c

```diff
@@ -791,12 +905,12 @@ static int adc_stm32_channel_setup(const struct device *dev,
 	}
 
 	if ((__LL_ADC_CHANNEL_TO_DECIMAL_NB(ADC_CHANNEL_TEMPSENSOR) == channel_cfg->channel_id) ||
-		(__LL_ADC_CHANNEL_TO_DECIMAL_NB(ADC_CHANNEL_VREFINT) == channel_cfg->channel_id)) {
+	    (__LL_ADC_CHANNEL_TO_DECIMAL_NB(ADC_CHANNEL_VREFINT) == channel_cfg->channel_id)) {
 		adc_stm32_set_common_path(dev);
 	}
 
 	adc_stm32_setup_speed(dev, channel_cfg->channel_id,
-				  acq_time_index);
+			      acq_time_index);
 
 	LOG_DBG("Channel setup succeeded!");
```

Comment:
This messages is printed for every channel and gets confusing.
```suggestion
	LOG_DBG("Channel %d setup succeeded!", channel_cfg->channel_id);
```

Reply:
done